.github/workflows/deploy.yml

```
@@ -1,16 +1,47 @@
         required: false
         default: 'Manual deployment from GitHub Actions'
 
 jobs:
+  determine-environment:
+    runs-on: ubuntu-latest
+    outputs:
+      environment: ${{ steps.set-env.outputs.environment }}
+    steps:
+      - name: Determine environment
+        id: set-env
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:64: Double quote to prevent globbing and word splitting

(shellcheck)